products/value.js

```diff
@@ -1,8 +1,3 @@
-// data [number, quantities, value]
-// const p_val = [["1", 12, 6500],
-//   ["2", 24, 12000], 
-//   ["3", 48, 23000], 
-//   ["4", 96, 45000]];
 const p_val = [{"num":"1", "qty": 12, "value": 6500},
```

Comment:
key don't need double quote
{name: 'jack' } is good enough!

Reply:
you can access data like this too.

p_val[0].value

this way is more popular because of less code i guess...

Reply:
but if your key has -, then 

p_val[0].some-value

is not correct.

in that case, you can use

p_val[0]['some-value']

Reply:
just for knowledge